wmn-data.json

```diff
@@ -1839,6 +1860,17 @@
         "known" : ["evdokia", "irina"],
         "cat" : "social"
        },
+       {
+        "name" : "Folkd",
+        "uri_check" : "https://www.folkd.com/?app=core&module=system&controller=ajax&do=usernameExists&input={account}",

```

Comment:
Please consider using a different `uri_check` like `https://www.folkd.com/search/?q={account}&quick=1&type=core_members` as it gives the link to the profile of the person in the results.

Reply:
To reduce network traffic, it would be better to add this to the `uri_pretty` parameter.